src/data_provider/src/packages/packageLinuxParserHelper.h

```diff
@@ -44,7 +44,21 @@ namespace PackageLinuxHelper
             }
         }
 
-        if (!info.empty() && info.at("Status") == "install ok installed")
+        /*
+           According to dpkg documentation, the status of the package consists in three fields separated by spaces:
+           'SELECTION_STATE FLAG PACKAGE_STATE'.
+
+           SELECTION_STATE: the desired action to take by the package manager. It could be 'install', 'hold',
+                            'deinstall', 'purge', or 'unknown'.
+           FLAG: indicates if the package requires a reinstall or if no issues were found. It could be 'ok',
+                 or 'reinstreq'.
+           PACKAGE_STATE: this is the real status of package at this moment. It could be 'not-installed',
+                          'config-files', 'half-installed', 'unpacked', 'half-configured', 'triggers-awaited',
+                          'triggers-pending', or 'installed'.
+
+           We'll collect packages in any selection state, but not broken ('ok') and correctly installed.
```

Comment:
Hi @sebasfalcone !

I meant that a broken package contains the flag `reinstreq`, and a not broken one `ok`.
The `ok` word is in the middle because it's referring to the flags. The "correctly installed" part refers to the `installed` word. I can't move the `ok` to the end, because the current state isn't related to that word.

What about this?

```
We'll collect packages in any selection state, with 'ok' FLAG and 'installed' PACKAGE_STATE
```

Reply:
Go for it!